programs/dmatest/meta.yaml

```diff
@@ -0,0 +1,12 @@
+short_description: >
+
+description: >
+
+homepage:
+
+parameters:
+ runtime:
+ scale_type:
```

Comment:
thanks, these 2 parameters are not aligned to dmatest.yaml, maybe need rename to use iterations or update dmatest.yaml.